CLI tools/CurseForge to Packwiz-Modrinth.py

```diff
@@ -90,32 +95,45 @@ def main() -> int:
             toml.dump(pack_toml, f)
 
         # Packwiz import
-        os.system(packwiz_exe_path + " curseforge import \"" + cf_zip_path + "\"")
+        os.system(f'{packwiz_exe_path} curseforge import "{cf_zip_path}"')
         if hydrogen:
-            os.system(packwiz_exe_path + " remove hydrogen")
-            os.system(packwiz_exe_path + " mr install hydrogen")
+            os.system(f"{packwiz_exe_path} remove hydrogen")
+            os.system(f"{packwiz_exe_path} mr install hydrogen")
         if modrinth_overrides:
-            os.system(packwiz_exe_path + " remove entityculling")
-            os.system(packwiz_exe_path + " mr install entityculling")
-            
-    elif refresh_only: 
-        os.system(packwiz_exe_path + " refresh")
+            os.system(f"{packwiz_exe_path} remove entityculling")
+            os.system(f"{packwiz_exe_path} mr install entityculling")
+
+    elif refresh_only:
+        os.system(f"{packwiz_exe_path} refresh")
 
     # Copy fresh manifest/modlist to git
     if not is_legacy and not refresh_only:
         extract_file(cf_zip_path, "manifest.json", git_path + "CurseForge", "CurseForge manifest.json", "Git")
         extract_file(cf_zip_path, "modlist.html", git_path + "CurseForge", "CurseForge modlist.html", "Git")
 
-    # Export packwiz pack via mmc-export method
+    # Export Packwiz pack via mmc-export method
     if mmc_export_packwiz_export and not refresh_only:
         mmc_zip_root = str(Path(cf_zip_path).parents[0])
         mmc_zip_path = mmc_zip_root + "\\Fabulously Optimized " + pack_version + ".zip"
         packwiz_config = git_path + "Packwiz\\mmc-export.toml"
 
-        cmd = f'mmc-export -i "{mmc_zip_path}" -f packwiz --modrinth-search loose -o "{mmc_zip_root}" -c "{packwiz_config}" -v {pack_version} --provider-priority Modrinth CurseForge Other --scheme mmc_export_packwiz_output'
-        os.system(cmd)
-
-        packwiz_zip_path = Path(mmc_zip_root) / "mmc_export_packwiz_output.zip" 
+        os.system(

```

Comment:
I think the command merging broke here, as I got the following error
> mmc-export: error: argument cmd: invalid choice: 'Optimized' (choose from 'gh-login', 'gh-logout', 'purge-cache')

and the rest failed. 

The command it joined was 

`"mmc-export -i 'c:\\Users\\Madis\\Desktop\\Fabulously Optimized 5.5.0-beta.4.zip' -f packwiz --modrinth-search loose -o 'c:\\Users\\Madis\\Desktop' -c 'C:\\Users\\Madis\\Documents\\GitHub\\fabulously-optimized\\Packwiz\\mmc-export.toml' -v 5.5.0-beta.4 --provider-priority Modrinth CurseForge Other --scheme mmc_export_packwiz_output"`

cc @RozeFound

Reply:
How does in even brakes? It outputs basically the same thing as the old string-merging code 
shlex output: ```mmc-export -i 'c:\Users\Madis\Desktop\Fabulously Optimized 5.5.0-beta.4.zip' -f packwiz --modrinth-search loose -o 'c:\Users\Madis\Desktop' -c 'C:\Users\Madis\Documents\GitHub\fabulously-optimized\Packwiz\mmc-export.toml' -v 5.5.0-beta.4 --provider-priority Modrinth CurseForge Other --scheme mmc_export_packwiz_output```
old method output: ```mmc-export -i "c:\Users\Madis\Desktop\Fabulously Optimized 5.5.0-beta.4.zip" -f packwiz --modrinth-search loose -o "c:\Users\Madis\Desktop" -c "C:\Users\Madis\Documents\GitHub\fabulously-optimized\Packwiz\mmc-export.toml" -v 5.5.0-beta.4 --provider-priority Modrinth CurseForge Other --scheme mmc_export_packwiz_output```
The only difference is type of quotes

Reply:
Oh, the type of quotes really does makes a difference on windows. My bad.

Reply:
@RozeFound yep, it does. That's what Python docs say:
> Warning: The shlex module is only designed for Unix shells.
> The `quote()` function is not guaranteed to be correct on non-POSIX compliant shells or shells from other operating systems such as Windows.

Waiting for your fix or revert :slightly_smiling_face: